include/crab/domains/union_find_domain.hpp

```diff
@@ -207,82 +198,82 @@ class union_find_domain {
 
   void get_all_members(std::vector<element_t> &out) const {
     out.reserve(m_parents.size());
-    for (auto &kv: m_parents) {
+    for (auto &kv : m_parents) {
       out.push_back(kv.first);
     }
   }
 
   // 1. Ensure that left and right have the same equivalence classes.
   // 2. Apply component-wise the join or the widening.
-  union_find_domain_t join_or_widening(const union_find_domain_t &o, bool is_join) const {
+  union_find_domain_t join_or_widening(const union_find_domain_t &o,
+                                       bool is_join) const {
     crab::CrabStats::count("union_find.count.join_or_widening");
     crab::ScopedCrabStats __st__("union_find.join_or_widening");
-    
+
     if (is_bottom()) {
       return o;
     } else if (o.is_bottom()) {
       return *this;
-    } else if (is_top() | o.is_top()) {
+    } else if (is_top() || o.is_top()) {
```

Comment:
this is important

Reply:
Thanks @priyasiddharth! . I'll do the fixes by myself to separate all the noise from clang-format.